code/modules/modular_computers/file_system/programs/engineering/shields_monitor.dm

```diff
@@ -0,0 +1,110 @@
+/datum/computer_file/program/shields_monitor
```

Comment:
Is this a non-changing version of shield control program?

Reply:
yes and it disabled

Reply:
ive deleted it now